src/components/radios/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't think this is meant to be added? None of the other components have package.json files in the `src` directory.

Reply:
Yes indeed. I've taken it out.